internal/service/apprunner/vpc_connector_test.go

```diff
@@ -233,6 +227,21 @@ resource "aws_apprunner_vpc_connector" "test" {
 `, rName, tagKey1, tagValue1)
 }
 
+func testAccVPCConnectorConfig_tags2(rName string, tagKey1 string, tagValue1 string, tagKey2 string, tagValue2 string) string {
+	return testAccVPCConnectorConfig_base(rName) + fmt.Sprintf(`
```

Comment:
You can also use `acctest.ConfigCompose()` here instead of + for combining the strings.

Reply:
Updated to use ConfigCompose